include/onnxruntime/core/optimizer/graph_transformer_utils.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer.h"
+#include "core/optimizer/rewrite_rule.h"
+
+namespace onnxruntime {
+
+namespace transformerutils {
+/* Validates whether level can be mapped to a valid TransformerLevel enum
+*/
+Status ValidateTransformerLevel(unsigned int level);
+
+/* Sets bit mask for transformers enabled based on the level provided
+*  Also populates a convenience list of all transformers  
+*/
+void SetTransformerContext(const uint32_t& level, uint32_t& levels_enabled, 
+                           std::vector<TransformerLevel>* all_levels = nullptr);
```

Comment:
Some more documentation is needed here. What is level, levels_enabled, all_levels (do we need that last one?)?